src/app/query/bitcoin/ordinals/use-next-fresh-taproot-address.query.ts

```diff
@@ -6,15 +6,15 @@ import { createCounter } from '@app/common/utils/counter';
 import { UtxoResponseItem } from '@app/query/bitcoin/bitcoin-client';
 import { getTaprootAddress } from '@app/query/bitcoin/ordinals/utils';
 import { useCurrentAccountIndex } from '@app/store/accounts/account';
-import { useTaprootAccountKeychain } from '@app/store/accounts/blockchain/bitcoin/taproot-account.hooks';
+import { useTaprootAccount } from '@app/store/accounts/blockchain/bitcoin/taproot-account.hooks';
 import { useBitcoinClient } from '@app/store/common/api-clients.hooks';
 import { useCurrentNetwork } from '@app/store/networks/networks.selectors';
 
 // ts-unused-exports:disable-next-line
 export function useNextFreshTaprootAddressQuery(accIndex?: number) {
   const network = useCurrentNetwork();
   const currentAccountIndex = useCurrentAccountIndex();
-  const account = useTaprootAccountKeychain(accIndex ?? currentAccountIndex);
+  const account = useTaprootAccount(accIndex ?? currentAccountIndex);
```

Comment:
Here we are maintaining this function that isn't used anywhere. 

I see a comment saying `// TODO: Reuse later for privacy mode` for it  but IMHO we would be better served to remove it in a commit with the issue number for when we will do `privacy mode`.